pkg/rtc/room_test.go

```diff
@@ -737,6 +735,10 @@ func newRoomWithParticipants(t *testing.T, opts testRoomOpts) *Room {
 		&livekit.Room{Name: "room"},
 		nil,
 		WebRTCConfig{},
+		config.RoomConfig{
+			EmptyTimeout:     5 * 60,
+			DepartureTimeout: 1,
```

Comment:
should we change the default to be higher? i.e. 20s?

Reply:
This is a test file. Config.go has the default of 20s

Reply:
oops.. misread.. myb